react/features/base/participants/actions.js

```diff
@@ -393,6 +369,10 @@ export function participantRoleChanged(id, role) {
  * }}
  */
 export function participantUpdated(participant = {}) {
+    if (participant.name) {
+        participant.name = participant.name.substr(0, MAX_DISPLAY_NAME_LENGTH);
+    }
```

Comment:
I'm generally not for mutating objects that get passed in. I'm pointing it out as a personal style thing, not as a requested change.

Reply:
Good point, thanks!